code-challenges.test.js

```diff
@@ -37,11 +71,49 @@ const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// concat arrays so that they become one and set to a new variable
+//declare new variable and use .filter method to return a subset of just numbers
+//create an HOF that maps over the array of numbers and divides each by 3
+//**** I'm sure there was a simpler way of going about it and I am curious to see what that'd look like- the refactored version */
+//the test did not pass although I did get the expected output. I think I might have put too many console logs 
```

Comment:
Same thing here, the jest test is specifically testing for a function with the name of `onlyRemain`, but it can't find it anywhere since it is never declared, which will cause a ReferenceError.